x-pack/plugins/apm/public/components/shared/suggestions_select/index.tsx

```diff
@@ -48,11 +55,16 @@ export function SuggestionsSelect({
     (callApmApi) => {
       return callApmApi('GET /internal/apm/suggestions', {
         params: {
-          query: { field, string: searchValue },
+          query: {
+            fieldName,
+            fieldValue: searchValue,
+            start: start ?? moment().subtract(24, 'h').toISOString(),
```

Comment:
Preferably we don't have defaults here either, but always require it and let the consuming component set it.